html-folder/alerts.html

```diff
@@ -85,7 +85,9 @@ <h2>Get Started</h2>
                                 <li class="aside-ul-li margin-bot-med">
                                     <a href="./ratings.html" class="aside-link link-btn ">Ratings</a>
                                 </li>
-
+                                <li class="aside-ul-li margin-bot-med">
+                                    <a href="./ratings.html" class="aside-link link-btn ">Slider</a>
```

Comment:
please remove the extra space after last class name link-btn.

Reply:
Done with this